kubeapps.jsonnet

```diff
@@ -3,17 +3,34 @@ local kubecfg = import "kubecfg.libsonnet";
 
 local host = null;
 local tls = false;
+local kubeless = import "kubeless.jsonnet";
+local ssecrets = import "sealed-secrets.jsonnet";
+
+local labels = {
+  metadata+: {
+    labels+: {
+      "created-by": "kubeapps"
+    }
+  }
+};
+// Some manifests are nested deeper than the root (e.g. dashboard.api.deploy)
+// so we need to make sure we're only applying the labels to objects that have
+// the manifest key
+local labelify(src) = if std.objectHas(src, "metadata") then src + labels else src;
```

Comment:
we could probably also do this recursively if it's nested deeper, which would avoid the need for https://github.com/kubeapps/manifest/pull/26/files#diff-f60a9b2633285b37b0c57323762556d6R107 but it's not a big deal as this is only deeply nested in the dashboard manifests